### PR TITLE
feat(animation): animation setup

### PR DESCRIPTION
**Pull request by tam-kis**

### CODE REVIEW

File: _includes/js/main.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { Slider } from "./slider.js";
 import { Locations } from "./locations";
+import { SlideIn } from "./animate-slidein";
 
-(() => {
+window.onload = () => {
```
Comment (Wandkleister) [inline _includes/js/main.js:5]: Please don't use an Event Listener here. This caused loading problems and was fixed this morning on the development branch. If you need to listen to the load event of the window object, you'll need to do that in your component.
```suggestion
(() => {
```

File: _includes/js/main.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
 	new Locations();
 	new Slider();
-})();
+	new SlideIn();
+};
```
Comment (Wandkleister) [inline _includes/js/main.js:9]:
```suggestion
})();
```

**Review by Wandkleister — CHANGES_REQUESTED** [review]

File: styles/includes/_footer.scss [hunk 3]
```diff
@@ -52,7 +52,7 @@
 
 .accenture-brand {
 	position: absolute;
-	top: -4.9rem;
+	top: -5.9rem;
```
Comment (luismeyer) [inline styles/includes/_footer.scss:55]: this creates a gap between the footer and the accenture thingy. i this wanted?

Comment (tam-kis) [reply styles/includes/_footer.scss]: This change now fixes the overlapping

**Review by Wandkleister — APPROVED** [review]

**Pull request merged**